src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+    public static int numberOfTasks = 0;
+    public static Task[] all = new Task[100];
```

Comment:
perhaps the 100 can be made into a constant? So that it is more meaningful instead of just having 100 as a magic number